src/main/kotlin/com/msd/robot/application/SuccessEventSender.kt

```diff
@@ -23,25 +21,29 @@ class SuccessEventSender(
     val robotDomainService: RobotDomainService
 ) {
 
-    /**
-     * Send the event after executing a successful movement.
-     */
-    fun sendMovementEvent(
-        robot: Robot,
-        cost: Int,
-        transactionUUID: UUID
-    ): UUID {
-        return eventSender.sendEvent(
-            MovementEventDTO(
-                true,
-                "Movement successful",
-                robot.energy,
-                planetMapper.planetToPlanetDTO(robot.planet, cost, PlanetType.DEFAULT), // TODO planet type?
-                robotDomainService.getRobotsOnPlanet(robot.planet.planetId).map { it.id }
-            ),
-            EventType.MOVEMENT,
-            transactionUUID
-        )
+    fun sendAllMovementEvents(robotPlanetPairs: MutableMap<MovementItemsUsageCommand, Pair<Robot, GameMapPlanetDto>>) {
```

Comment:
What is this method for? It just sends a MovementEvent and an ItemMovementEvent. Where is the neighboursEvent? Why don't you use the existing method that sends MovementEvents? This method should also probably be named `sendItemMovementEvents`.

Reply:
it uses the existing method, but I'll rename it